packages/TelegramClientTests-Core.package/TCTCCoreResource.class/instance/core..st

```diff
@@ -0,0 +1,3 @@
+accessing
+core: anObject
+	core := anObject
```

Comment:
linter fails, parameter should have a meaningful name and blank line after first the first line is needed